### PR TITLE
Chat messages hyperlink. Handling a case when a chat user deleted an account.

### PR DESCRIPTION
**Pull request by BartDevTWG**
Adding ExternalLink component.
Linkifying chat messages. 
Zrzut ekranu 2021-01-28 o 14 35 37

Handling a case when there is an error (e.g. chat user deleted an account, we get 404), to prevent the app crash.

## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate).

### CODE REVIEW

File: src/chat/components/Message.tsx [hunk 1]
```diff
@@ -67,6 +69,12 @@ export interface MessageProps {
   created: string | Date;
 }
 
+const componentDecorator = (href: string, text: string) => (
+  <ExternalLink href={href}>
+    <>{text}</>
```
Comment (alex-blair) [inline src/chat/components/Message.tsx:74]: Is the fragment needed?

File: src/components/ExternalLink.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  const classes = useStyles();
+  return (
+    <a
+      className={classes.link}
+      target="_blank"
+      rel="nofollow noreferrer noopener"
+      href={href}
+    >
+      {children}
+    </a>
```
Comment (alex-blair) [inline src/components/ExternalLink.tsx:29]: Nice! I love that you've made this its own component

**Review by alex-blair — APPROVED** [review]
Looks good! 💪

**Review by MichaelAmoabengVaulitions — APPROVED** [review]
LGTM!

File: src/chat/components/Message.tsx (hunk 1 above)
Comment (BartDevTWG) [reply src/chat/components/Message.tsx]: I've changed it 🙂

**Pull request merged**